ports/boost-modular-build-helper/CMakeLists.txt

```
@@ -117,10 +117,23 @@ string(APPEND CFLAGS "${COMPILEFLAGS}")
 foreach(var CXXFLAGS CFLAGS LDFLAGS ARFLAGS)
     string(REPLACE [[\]] [[\\]] ${var} "${${var}}")
     string(REPLACE [["]] [[\"]] ${var} "${${var}}")
+    if(WIN32)
+        # env variables need extra '' to be not expanded early
+        string(REGEX REPLACE [[(%[^%]+%)]] [['\1']] ${var} "${${var}}")

```

After testing again. This seems to be the correct fix:
```suggestion
        string(REGEX REPLACE [[(%[^%]+%)]] [[%\1%']] ${var} "${${var}}")
```

Somehow my pdb copy script still does not find the pdb which threw me off. After manually inspecting the dll with dumpbin I see the correct pdb path with this change.

---

This makes `%_PDB%` into `%%_PDB%%` which is more like what i would have expected.